ark/type/methods/base.ts

```diff
@@ -42,12 +42,25 @@ import type { instantiateType } from "./instantiate.ts"
 interface Type<out t = unknown, $ = {}>
 	extends Callable<(data: unknown) => distill.Out<t> | ArkErrors> {
 	[inferred]: t
+	/**
+	 * Generic argument. May be used for type reconstruction.
```

Comment:
The top-level generic parameter accepted by the `Type`. Potentially includes morphs and subtype constraints not reflected in the types fully-inferred input (via `inferIn`) or output (via `infer` or `inferOut`).